### ISSUE TITLE
add guide on how to install Ionic Core with other frameworks

### ISSUE THREAD

**Issue by liamdebeasi** [ionic-team/ionic-docs#2105]
**URL**
https://ionicframework.com/docs/intro/cdn

**What is missing or inaccurate about the content on this page?**
Ionic Core does work with any library but may lack form/routing integrations. At the very least, we should show how to install @ionic/core, import and use components, and import any required CSS.

This guide should follow the CE build rather than lazy loaded build for optimized tree shaking. We need to cover the followings things:

1. How to install @ionic/core
2. How to properly import and register components
3. How to import the required CSS
4. How to add the `ion-ce` class to show the app.

**Comment by p-baum** [comment]
Can we please change the title such that it contains the words "vanilla javascript" (or words to that effect).  Ionics insistence that I use a framework is frankly abhorrent.  I will nevertheless persist with my efforts to get it working without documentation despite feeling entirely unwelcome (for a short time at least).

**Comment by p-baum** [comment]
Still waiting for this.  Please tell me how to import and use the components without a framework.

I intend to bundle with vite and use server side partials

**Comment by gama410** [comment]
I agree with this issue. I ended up finding an example repo for a vanilla typescript build but it's not official. Ionic is already very good by itself and for small apps, a vanilla build is a viable (and valid) option.
Also, the CLI creating only framework-bound projects makes it very hard to know if ionic should really be used by itself...

@p-baum  For reference, here is the project that got me started: https://github.com/supertorpe/ionic-vanilla-ts

**Comment by isabo** [comment]
These instructions were very helpful: https://www.npmjs.com/package/@ionic/core#custom-elements-build-experimental

The only thing missing is that we need to execute `document.documentElement.classList.add('ion-ce');` which I got from https://github.com/ionic-team/ionic-framework/issues/24252#issuecomment-976587787. Thank you @liamdebeasi !

**Comment by DamianoP** [comment]
> Still waiting for this. Please tell me how to import and use the components without a framework.
> 
> I intend to bundle with vite and use server side partials

yea, would be very useful!
